exercises/concept/gross-store/.docs/introduction.md

```diff
@@ -8,12 +8,8 @@ Syntactically, `map` looks like this:
 map[KeyType]ElementType
 ```
 
-_`KeyType` must be any [comparable type][gospec-comparable], while `ElementType` can be any valid type in go, which means you can store anything from primitive variable to a slice._
```

Comment:
Could you please apply the same changes you did for this introduction file to the maps concept `introduction.md`.